app/Models/Message.php

```diff
@@ -56,6 +56,26 @@ public function setProTipAttribute($value)
     }
 
     /**
+     * Set the reportback_id attribute for the message.
+     *
+     * @param string  $value
+     */
+    public function setReportbackIdAttribute($value)
+    {
+        $this->attributes['reportback_id'] = ! empty($value) ? $value : null;
```

Comment:
Maybe switch these to match the `setProTipAttribute()` method above so it uses the same syntax, checking for empty, instead of `!` empty:

```php
public function setProTipAttribute($value)
    {
        $this->attributes['pro_tip'] = empty($value) ? null : $value;
    }
```

Or switch the syntax of `setProTipAttribute()`; whatever is the pref, just make consistent :)